contract/models/account_analytic_account.py

```diff
@@ -53,6 +54,22 @@ class AccountAnalyticAccount(models.Model):
     create_invoice_visibility = fields.Boolean(
         compute='_compute_create_invoice_visibility',
     )
+    generate_before = fields.Integer(
+        string="Generate before",
```

Comment:
I think the system will automatically use the name of the field so we don't need to define the string for the fields again, so please remove the string.